### PR TITLE
Input Validation Text

### LINKED ISSUE #332: Login inputs
## Acceptance Criteria

- [ ] Input validation text should be grammatically correct (no camel case)
- [x] Input labels should animate back into place correctly when deselected with no input.
- [x] Input labels shouldn't "bounce" between empty and filled state.

## Context

This appears to only be affecting the login inputs (give inputs seem ok but validation isn't handled consistently everywhere) but all platforms seem to suffer. Validation seems to be showing the variable name. When you select an input and deselect without inputing anything the label animates to an incorrect position. Labels seem to "bounce" between empty and filled input state. 

#### validation text
screenshot_1518623515
#### label placement
inputs
#### "bouncing"
jumpy

### PR DESCRIPTION
**Pull request by richarddubay**
Fixes #332 

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [ ] Review new test logic

### CODE REVIEW

**Comment by richarddubay** [comment]
screenshot 2018-02-28 15 07 24

File: src/@ui/forms/SignUpForm.js [hunk 1]
```diff
@@ -16,13 +16,16 @@ import Status from './FormStatusText';
...
+    <ButtonLink onPress={() => Linking.openURL('https://newspring.cc/terms')}>
+      terms of service
+    </ButtonLink>.
   </LabelText>
 );
 
 const UserExistsStatus = withRouter(({ history }) => (
   <Status error>
-    {'A user already exists with that email, but the password doesn\'t match. Would you like to '}
+    {"A user already exists with that email, but the password doesn't match. Would you like to "}
```
Comment (IsaacHardy) [inline src/@ui/forms/SignUpForm.js]: We need to do something about our prettier settings.

**Comment by IsaacHardy** [comment]
@burkeshartsis Everything looks good on my end. Wanted to make sure your eyes viewed it before we merge.

**Comment by IsaacHardy** [comment]
@richarddubay This is good to merge.

**Pull request merged**